lib/pgBackRest/Backup/Backup.pm

```diff
@@ -183,25 +185,27 @@ sub tmpClean
     (
         $strOperation,
         $oFileLocal,
+        $strBackupLabel,
         $oManifest,
         $oAbortedManifest
     ) =
         logDebugParam
     (
         __PACKAGE__ . '->tmpClean', \@_,
         {name => 'oFileLocal', trace => true},
+        {name => 'strBackupLabel', trace => true},
         {name => 'oManifest', trace => true},
         {name => 'oAbortedManifest', trace => true}
     );
 
-    &log(DETAIL, 'clean backup temp path: ' . $oFileLocal->pathGet(PATH_BACKUP_TMP));
+    &log(DETAIL, 'clean resumed backup path: ' . $oFileLocal->pathGet(PATH_BACKUP_CLUSTER, $strBackupLabel));
 
     # Get the list of files that should be deleted from temp
```

Comment:
1) Renamed to `resumeClean()`.
2) Updated.
3) Done.

I also searched for "temp" and didn't find any other instances.